app/bundles/SmsBundle/EventListener/SmsSubscriber.php

```diff
@@ -123,6 +132,11 @@ public function onDelete(SmsEvent $event)
      */
     public function onTokenReplacement(TokenReplacementEvent $event)
     {
+        // Disable trackable urls
+        if ($this->smsHelper->getDisableTrackableUrls()) {
+            return;
```

Comment:
I am not sure we can simply return here - we need to replace Lead tokes too?

Reply:
Right, I will update it. Thanks